xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -389,9 +395,17 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
             context.coordinator.isUIViewUpdating = false
         }
 
-        // Update text
-        if view.text != state.text {
+        // Update/re-render text if text changed or font size
+        // preferences changed.
+        if (
+            view.text != state.text ||
+            self.bodySize != context.coordinator.renderer.bodySize
+        ) {
             SubtextTextViewRepresentable.logger.debug("updateUIView: set text")
+            // Set body size on renderer. This costs nothing, and makes sure
+            // that the next text render will have the new body size if the
+            // body size preferenced changed.
+            context.coordinator.renderer.bodySize = bodySize
```

Comment:
Oof, of course we have to do this. I guess it's better to have solved this now before we get any further into the the transclude block work.